js/angular-crmMailing/directives.js

```diff
@@ -28,6 +28,7 @@
           });
           scope.crmMailingConst = CRM.crmMailing;
           scope.ts = CRM.ts(null);
+          scope.helpFile = 'CRM/Mailing/Form/Upload';
```

Comment:
BTW I'm going to change this file path. Instead of using the existing civimail help files (which are slated for demolition) I'll create a new one somewhere...
todo: where?